### ISSUE TITLE
Request for pretrained weights for other models compared in your paper

### ISSUE THREAD

**Issue by KoubaPetr** [open-mmlab/mmaction2#954]
Hello, congratulations on the amazing work and the great paper! What I especially enjoyed about your paper was the methodical comparison to GCNs and the evaluation of role of quality of the skeletons.

I would like to study how your method and the GCNs compare even further and it would be a great help if you could provide me with the pretrained weights you got for the MS-G3D when training on the HRNet 2D skeletons from the Kinetics dataset (the weights which led you to the accuracy 44.9% for the GCN).

Thank you very much in advance for any reply.

**Comment by dreamerlin** [comment]
It seems this issue is not for MMAction2

**Comment by KoubaPetr** [comment]
Yes, not directly, it is however the result of paper. Any idea where I should aim this request?

Thanks a lot.

**Comment by innerlee** [comment]
Email the authors of the paper directly would be great

**Comment by KoubaPetr** [comment]
Ok, I will give it a try, Thank you!

**Comment by kennymckormick** [comment]
We will release the Kinetics-related experiments (checkpoints, annotations) in the future. Currently, we have only released checkpoints of FineGYM & NTURGB-D.

**Comment by priyamdey** [comment]
Hi. I'm new to this repository. I was planning to use the Kinetics-pretrained RN50 backbone ([this](https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/tsn/tsn_r50_320p_1x1x8_100e_kinetics400_rgb.py)). Is there a link available for download? Thanks.
